test/test_models.py

```diff
@@ -449,7 +449,12 @@ def DeleteAll(KEYS):
     key.delete()
 
 
-class ModelsTest(unittest2.TestCase):
+class ModelsTest(unittest.TestCase):
+
+  def setUp(self):
+    app_engine_test_utils.activate_app_engine_testbed()
+    app_engine_test_utils.clear_ndb_cache()
```

Comment:
Is there any reason to have one of these calls and not the other?  could combine them for simplicity and to avoid errors.  activate_app_engine_testbed_and_clear_ndb_cache()

Reply:
The docs are a bit unclear but I separated because it was suggested clearing the cache between all tests is best practice whereas the testbed stuff can be done once per test class. That said it's super fast for us so I just did both always to ensure a clean starting state and we should probably make it one function so the utils api is cleaner. Will do!